lib/dependabot/file_fetchers/java/maven.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require "dependabot/file_fetchers/base"
+
+module Dependabot
+  module FileFetchers
+    module Java
+      class Maven < Dependabot::FileFetchers::Base
+        def self.required_files_in?(filenames)
+          (%w(pom.xml) - filenames).empty?
+        end
+
+        def self.required_files_message
+          "Repo must contain a pom.xml."
+        end
+
+        private
+
+        def fetch_files
+          fetched_files = []
+          fetched_files << pom
```

Comment:
TODO: are the ay additional files we might need to load, or will everything be in the `pom.xml`?

Reply:
A Maven project can have 1 or potentially many nested `pom.xml` files for various submodules

Reply:
I think I'll target just one aggregator POM initially, to get to know the codebase and make progress. If everything goes OK, I'll go back and revisit that part.